examples/madry_lab_challenges/attack_model.py

```diff
@@ -0,0 +1,96 @@
+"""Runs CleverHans attacks on the Madry Lab MNIST challenge model
+
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import time
+
+import tensorflow as tf
+from tensorflow.examples.tutorials.mnist import input_data
+from tensorflow.python.platform import app
+from tensorflow.python.platform import flags
+from madry_mnist_model import MadryMNIST
+from cleverhans.utils_tf import model_eval
+from cleverhans.attacks import FastGradientMethod
+from cleverhans.attacks import BasicIterativeMethod
+from cleverhans.utils_mnist import data_mnist
+
+
+FLAGS = flags.FLAGS
+
+
+def main(argv):
+    checkpoint = tf.train.latest_checkpoint(FLAGS.checkpoint_dir)
+
+    if checkpoint is None:
+        raise ValueError("Couldn't load checkpoint")
+
+    train_start = 0
+    train_end = 60000
+    test_start = 0
+    test_end = 10000
+    X_train, Y_train, X_test, Y_test = data_mnist(train_start=train_start,
+                                                  train_end=train_end,
+                                                  test_start=test_start,
+                                                  test_end=test_end)
+
+    assert Y_train.shape[1] == 10.
```

Comment:
That's apparently an oddity that's been copy-pasted around the tutorials all along. I fixed it both here and in the tutorials